docs/configuration.md

```diff
@@ -291,13 +295,28 @@ Format for networks is:
 ...
 ```
 
+```json
+...
+"networks": {
+  "NETWORK NAME": {
+    "Host": "ADDRESS",
+    "NetworkKey": "ACCESS NODE NETWORK KEY"    
+  }
+}
+...
+```
+
 ```json
 ...
 
 "networks": {
     "emulator": "127.0.0.1:3569",
     "mainnet": "access.mainnet.nodes.onflow.org:9000",
-    "testnet": "access.devnet.nodes.onflow.org:9000"
+    "testnet": "access.devnet.nodes.onflow.org:9000",
+    "testnetSecure": {
+        "Host": "access-001.devnet30.nodes.onflow.org:9001",
```

Comment:
Should this be lowercased now?

Reply:
https://github.com/onflow/flow-cli/pull/449/commits/05b28f3a8666def59064eebb64f01d7102dc1e6c